### PR TITLE
[DEV-11280] Position overlay menu below the button element

### PR DESCRIPTION
**Pull request by fgyimah**
https://github.com/prezly/slate/assets/43710420/be8d0ff7-3d71-453d-96f6-0548cb92e796

### CODE REVIEW

File: packages/slate-editor/src/modules/popper-options-context/types.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export interface Context {
     modifiers?: {
         preventOverflow?: PreventOverflowModifier['options'];
     };
-    placement?: 'left' | 'right';
+    placement?: 'left' | 'right' | 'bottom';
```
Comment (e1himself) [inline packages/slate-editor/src/modules/popper-options-context/types.ts:8]: Can we make it work like in the Figma mockups?

1. If the button is left-aligned, the menu is bound to the button bottom left corner.
2. If the button is centred, the menu is bound to the button bottom middle part.
3. If the button is right-aligned, the menu is bound to the button right corner.

Screenshot from 2023-08-15 11-50-57
Screenshot from 2023-08-15 11-50-53
Screenshot from 2023-08-15 11-50-45

Comment (e1himself) [reply packages/slate-editor/src/modules/popper-options-context/types.ts:8]: Checking.

Comment (e1himself) [reply packages/slate-editor/src/modules/popper-options-context/types.ts:8]: > image

Nope, this is not good enough. Let me see if I can fix it.

Comment (e1himself) [reply packages/slate-editor/src/modules/popper-options-context/types.ts]: dfc7ebb6395baa26a083ea90384f3d9a6723fde6

[Screencast from 2023-08-16 16-29-17.webm](https://github.com/prezly/slate/assets/370680/3a53e45f-1cdf-4d58-be7d-ffebcf6b4d8b)

Comment (fgyimah) [reply packages/slate-editor/src/modules/popper-options-context/types.ts]: Looks good!

**Review by e1himself — APPROVED** [review]

**Pull request merged**